Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Controller.cs

```diff
@@ -299,9 +329,9 @@ public override bool Use(float deltaTime, Character activator = null)
 #endif
                 }
             }
-            else
+            else if (!string.IsNullOrEmpty(output))
             {
-                item.SendSignal(new Signal("1", sender: user), "trigger_out");
+                item.SendSignal(new Signal(output, sender: user), "trigger_out");
             }
```

Comment:
Could be optimized by deactivating the component when the output is empty (would need to ofc reactivate if the `State` or the output changes).

Reply:
Addressed by not reactivating the component if `output` or `falseOutput` are changed to a blank signal, and by deactivating the component if the current output is blank while `IsToggle` is true.